src/luascript.cpp

```diff
@@ -340,6 +340,42 @@ int32_t LuaScriptInterface::loadFile(const std::string& file, Npc* npc /* = null
 	return 0;
 }
 
+std::string LuaScriptInterface::loadString(const std::string& string, const std::string& fileName)
+{
+	//loads string as a chunk at stack top
+	int ret = luaL_loadstring(luaState, string.c_str());
+	if (ret != 0) {
+		lastLuaError = popString(luaState);
+		return lastLuaError;
```

Comment:
Shouldnt this use the reportError method and return boolean?

Reply:
I could use reportError here yea, I basicly tried to mimic the loadFile version where it was basicly made like this.
For the returning as bool, no that wont work because I'm pushing the value as a string on that we can send the error message to the API